### ISSUE TITLE
Reproducing DQN Breakout result

### ISSUE THREAD

**Issue by BryanBo-Cao** [keras-team/keras-io#89]
Thanks for the efforts in writing the rl code examples!

I tried to reproduce the results by running the [deep n networks for Breakout](https://keras.io/examples/rl/deep_q_network_breakout/) in [my colab](https://drive.google.com/file/d/1bjKBdzaGSmE1X_KBpHZk6iI0D3tWpp6a/view?usp=sharing) and train my agent with RTX 2060 for a whole day, but the reward didn't increase even after 10 million frames:
```
running reward: 0.31 at episode 302, frame count 10000
running reward: 0.32 at episode 594, frame count 20000
running reward: 0.28 at episode 884, frame count 30000
...
running reward: 5.94 at episode 35070, frame count 2620000
running reward: 5.07 at episode 35122, frame count 2630000
running reward: 5.35 at episode 35166, frame count 2640000
...
running reward: 0.31 at episode 276916, frame count 11610000
running reward: 0.21 at episode 277225, frame count 11620000
running reward: 0.19 at episode 277536, frame count 11630000
```

My environment settings:
| OS/Driver/Lib  | Version |
| ------------- | ------------- |
| Ubuntu  | 18.04.4 LTS  |
| GPU Driver  | 450.36.06  |
| CUDA | 11.0 |
| Tensorflow | 2.2.0 |
| Keras | 2.3.1 |
Note that ```tf-nightly``` has been installed.

The [document](https://keras.io/examples/rl/deep_q_network_breakout/) mentions that 10 million frames should be sufficient. Did I miss something?

Any help would be appreciated!

**Comment by mlech26l** [comment]
Hi, 

there was a bug in the example, which may caused some issues during learning. 
In particular, the weights of the target model were not properly duplicated (the Q evaluation model and the target model always used the same weight values).

The code on the website has been updated. 

I have tested the code. After 10 million frames, the model achieved a score of around 10 points.

**Comment by BryanBo-Cao** [comment]
Thanks! @mlech26l 
Here is my result for the new version:
![image](https://user-images.githubusercontent.com/14010288/87203871-885d7100-c2d1-11ea-9f6f-20ebee46017d.png)
